iceoryx_examples/complexdata/README.md

```diff
@@ -116,7 +116,7 @@ handleInsertionReturnVal(sample->optionalList.push_front(nullopt));
 
 !!! note
     If you're not familiar with `optional`, please have a look at
-    [How optional and error values are returned in iceoryx](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/advanced/how-optional-and-error-values-are-returned-in-iceoryx.md#optional).
+    [How optional and error values are returned in iceoryx](https://iceoryx.io/latest/advanced/how-optional-and-error-values-are-returned-in-iceoryx/#optional).
```

Comment:
Shouldn't we link here to the markdown file in our github repo. Otherwise we can quickly stumble over invalid links when the files moved and are not yet updated.

Please never link to the website from within a markdown file, just link to the github repo or set a relative link.